src/sentry/search/events/builder/discover.py

```diff
@@ -275,6 +278,7 @@ def __init__(
         params: ParamsType,
         interval: int,
         top_events: list[dict[str, Any]],
+        snuba_params: SnubaParams | None = None,
```

Comment:
Should this be passed to `super().__init__`?

Reply:
woops good catch